docs/environment.yml

```diff
@@ -0,0 +1,10 @@
+name: tljh
+channels:
+  - conda-forge
+dependencies:
+- python=3.6
+- traitlets>=4.1
+- sphinx>=1.4, !=1.5.4
+- pip:
+  - recommonmark==0.4.0
```

Comment:
we don't need recommonmark :)

Reply:
oops! ok removing that line and merging...I may merge a couple more to get RTD working if something is off